build_docs.py

```diff
@@ -28,7 +28,7 @@
 import logging
 import logging.handlers
 from functools import total_ordering
-from os import readlink
+from os import readlink, EX_OK, EX_SOFTWARE
```

Comment:
These newly added attributes are not available on non-Unix platforms. Not sure if that's relevant TBH; since I don't know if we run these scripts on non-Unix platforms (mainly, Windows).

Reply:
I think a notice can be shown either in the README or in the code ("try... except ImportError" etc.) or both. This would probably save some time for people using Windows.

Reply:
I've wrapped the import in try-except block, adding fallback values.

(Trivia: Windows 10 lacks EX_OK despite docs claiming Windows supports this one exit code).

Side note: we run this script only on Linux, the script and Makefile in cpython Doc directory use `sed`, which is not present on Windows and not compatible with Mac-flavour sed. But I don't see the point to make potential usage on Windows more difficult in the future (by leaving this failing imports as they were).